uca-net-camera.c

```diff
@@ -411,8 +411,18 @@ uca_net_camera_get_property (GObject *object,
     priv = UCA_NET_CAMERA_GET_PRIVATE (object);
 
     /* handle net camera props */
-    if (property_id == PROP_HOST) {
-        g_value_set_string (value, priv->host);
+    switch (property_id) {
+        case PROP_HOST:
+            g_value_set_string (value, priv->host);
+            return;
+        case PROP_IS_RECORDING:
+            if (priv->client == NULL) {
+                g_value_set_boolean (value, FALSE);
+            }
+    }
+
+    if (priv->client == NULL) {
+        g_debug ("Not requesting property becuase connection has been already closed");
```

Comment:
> It is possible that object methods might be invoked after dispose is run and before finalize runs. GObject does not consider this to be a program error: you must gracefully detect this and neither crash nor warn the user, by having a disposed instance revert to an inert state.

From [the docs](https://docs.gtk.org/gobject/tutorial.html)

and notify the user of our class that this is happening.